contrib/pg_tde/t/011_unlogged_tables.pl

```diff
@@ -31,6 +31,7 @@
 
 PGTDE::append_to_result_file("-- kill -9");
 $node->kill9();
+sleep(3);    # sanitizers slow down the kill, let server die in peace
```

Comment:
Maybe I am missing something but how can `kill -9` be slow?

Reply:
Here is the thread where exact same problem discussed: https://www.postgresql.org/message-id/flat/1709793.1644523137%40sss.pgh.pa.us#c4caef6ad812ecaceeb5b711b02ee8f4

So as I understood `kill -9` could wait for program to terminate syscals/coredumps and it looks like ASAN doing something that slightly delays node death.

Reply:
Already fixed in main branch, so removed this for PR